spec/merge_helpers_spec.rb

```diff
@@ -25,7 +25,7 @@
       end
 
       it "has proper results for sample.rb" do
-        expect(subject[source_fixture("sample.rb")]).to eq([1, 1, 2, 2, nil, nil, 2, 2, nil, nil])
+        expect(subject[source_fixture("sample.rb")]).to eq([nil, nil, 2, 2, nil, nil, 2, 2, nil, nil])
```

Comment:
Why did you change the existing behavior?
This will occur a problem when we run the tests in parallel. Because covered line become a disabled line.